api/routers/hosts.py

```
@@ -1,54 +1,85 @@
+        "user": user,
+        "settings": get_settings(),
+        "ssh_host": await get_ssh_host(),
+        "all_users": all_users,
+        "server_deploy_key": server_deploy_key,
+    }
+    if error:
+        context["error"] = error
+    return templates.TemplateResponse(request, "hosts.html", context=context, status_code=status_code)
+
+
+async def _host_row_response(request: Request, db: AsyncSession, user: User, host_id: str, *, with_users: bool = False):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_host_row_response`

(ANN202)